specifyweb/frontend/js_src/lib/components/FormEditor/Editor.tsx

```diff
@@ -148,6 +150,7 @@ export function FormEditorWrapper(): JSX.Element {
             ? icons.switchVertical
             : icons.switchHorizontal}
         </Button.Small>
+        <UseLabelsSchema />
```

Comment:
(optional) could you put a `<span className="flex-1 -ml-4" />` here to put the checkbox on the side? does that look better or not? does it lead to better UX (i.e it may look weird to have button to close to the checkbox)

Reply:
We tried and it was decided to keep the check box on the left next to the switch button :)